dts/bindings/rtc/st,stm32-rtc.yaml

```diff
@@ -18,3 +21,9 @@ properties:
     enum:
       - 1
       - 512
+
+  alarms-count:
+    description: |
+      Number of alarms supported by STM32 RTC device.
+      Most of STM32 MCU series have 2 RTC alarms, A & B.
+      Defaults to 0 when not declared in devicetree.
```

Comment:
either put a default: value here or make it required: true

Reply:
There is already a default value in `rtc-device.yaml`.

Reply:
ok